test/global.test.ts

```diff
@@ -11,6 +11,42 @@ import * as vscode from 'vscode';
 import { TestOutputChannel } from 'vscode-azureextensiondev';
 import { ext, getRandomHexString, getTemplateProvider, parseError, TemplateProvider, TemplateSource, TestUserInput } from '../extension.bundle';
 
+export namespace constants {
+    export const testOutputName: string = 'testOutput';
+}
+
+// The root workspace folder that vscode is opened against for tests
+let testRootFolder: string;
+
+export function getTestRootFolder(): string {
```

Comment:
I'd prefer to move this into `createAzureResource.test.ts`. Mostly because that's the only place using it and `testRootFolder` is named very similar to `testFolderPath` below which is pretty confusing.

As a side note: `createAzureResource.test.ts` is getting kinda big and I want to split that into separate files eventually, but we should do that in a separate PR all by itself